src/modules/endpoint/sagas/loadCityContent.js

```diff
@@ -81,7 +83,7 @@ export default function * loadCityContent (
 
   const [categoriesMap, events] = yield all([
     call(loadCategories, newCity, newLanguage, dataContainer, shouldUpdate),
-    call(loadEvents, newCity, newLanguage, dataContainer, shouldUpdate)
+    call(loadEvents, newCity, newLanguage, cityModel.eventsEnabled, dataContainer, shouldUpdate)
```

Comment:
Why do we have to set events to an empty array if events are disabled? Can't we just return an empty array here if disabled?

Reply:
The data container has to be set as well, since other sagas might as well query events from the data container (and should receive an empty array).

Reply:
Might query or are querying?

Reply:
Are querying